apps/cowswap-frontend/src/legacy/state/application/reducer.ts

```diff
@@ -40,22 +40,10 @@ export interface MetaTxPopupContent {
 export type WarningPopupContent = { warning: string }
 
 export enum ApplicationModal {
-  ADDRESS_CLAIM,
-  BLOCKED_ACCOUNT,
-  DELEGATE,
-  CLAIM_POPUP,
-  MENU,
```

Comment:
it must have felt good to delete all this

Reply:
![image](https://github.com/cowprotocol/cowswap/assets/7122625/e140d476-a935-45d9-b8fe-82817ca7972c)